example_files/example_lint_cfg.README

```diff
@@ -9,8 +9,9 @@
 // your modifications
 
 {
-    // if set to 'false', will lint indirectly imported files as well
-    "direct_only": true,
+    // if set to 'false', will parse and lint imported files as well
```

Comment:
With this change, this is no longer a lint option and should be passed as either a command-line argument or a server configuration. I think this lines up with your desire to have this run only from CLI mode anyway.

I would recommend making it a command-line flag (probably with a similar one set on the DFA so it can pick its mode as well)

Reply:
Done